src/ui.rs

```diff
@@ -87,12 +87,12 @@ fn create_components(left: &Box, centered: &Box, right: &Box) {
         // Identifier example: `left-label_ABC` <= `left-label` is the IDENTIFIER, `ABC` is the NAME.
         let identifier = identifiers[0];
 
-        // Grabs the widget alignment.
-        let widget_alignment = key
-            .split(ALIGNMENT)
-            .next()
-            .expect("[ERROR] Invalid widget alignment!\n")
-            .to_uppercase();
+        // Grabs widget alignment and widget type from the identifier separated by '-̈́'.
```

Comment:
`-̈́` ? You may wanna put in `-`, seems like your keyboard had a small seizure xd

Reply:
huh, weird, alacritty doesn't even render it differently, no idea how I did it lol
![image](https://user-images.githubusercontent.com/32438954/198829390-9834000b-239b-46ab-81db-ab1c1d94442c.png)